suites/upgrade/infernalis/older/3-upgrade-sequence/upgrade-osd-mon-mds.yaml

```diff
@@ -0,0 +1,44 @@
+meta:
+   - desc: |
+      upgrade mon a and b
+      restart order 
+      osd.0(1,2,3,4,5) wait 30 sec between restarts
```

Comment:
same as previous comments - simplify restart order to 'restart osds, monitors, then mds'